app/views/site/coc.html.haml

```diff
@@ -11,34 +14,172 @@
     Contact [help@indyhackers.org](mailto:help@indyhackers.org).
 
     If you want to chat about something/report a violation you can contact any of the current board members:
```

Comment:
I would suggest moving the code of conduct to the top of this page, and moving the board members below it. If we're linking to the code of conduct than the user will likely be more interested in reading that rather than knowing who to contact about it.